multiqc/utils/search_patterns.yaml

```
@@ -876,3 +876,6 @@ verifybamid/selfsm:
 whatshap/stats:
   contents: "#sample	chromosome	file_name	variants	phased	unphased	singletons"
   num_lines: 1
+mosaicatcher:
+  contents: "# sample:  Sample (has multiple cells)"
```

For extra efficiency:

```suggestion
  contents: "# sample:  Sample (has multiple cells)"
  num_lines: 1
```

---

Thanks for the improvement :)